pkg/errors/errors.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Error is a concrete error implementation.
+package errors
+
+type Error string
+
+// Error satisfies the error interface.
+func (e Error) Error() string {
+	return string(e)
+}
+
+const (
+	ErrInvalidJustification = Error("invalid justification")
+	ErrInternal             = Error("internal error, unable to perform jira validation")
+)
```

Comment:
No need to make it more complicated than the following. Plus, I'm not sure why internal error should be its own type. Anything that's not an "invalid" error is an internal error?

```suggestion
var (
	ErrInvalidJustification = fmt.Errorf("invalid justification")
)
```